### ISSUE TITLE
Remove linkchecker from rake & install Travis

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.org#549]
[Integrations are no longer supported, and will go away entirely in January 2019](https://developer.github.com/v3/guides/replacing-github-services/).

**Comment by amandavisconti** [comment]
It looks like we can just remove the integration, and add the [Travis GH app](https://github.com/apps/travis-ci). Will try that now.

**Comment by amandavisconti** [comment]
**Zero rush** on this, but @walshbr I think we might need you to visit travis-ci.org and deactivate any connection to ScholarsLab repos (or at least scholarslab/scholarslab.org)? (The integration is being sunsetted, but I think we can't get the GH app version of Travis running until the old thing's deactivated?) Thank you!

**Comment by walshbr** [comment]
I did this, @amandavisconti, but it looks like it's still giving that warning and not letting us migrate for some reason. Might be worth taking a look together at the various settings when we're in office sometime? Or maybe looking from your side and trying to migrate the thing from the screen you were looking at.

**Comment by walshbr** [comment]
Also we have travis accounts running for a couple dozen repos - are you sure you want me to wholesale bag all of them? Feels like someone should go through and migrate each of them one by one, but I know nothing about any of the other projects.

**Comment by amandavisconti** [comment]
@walshbr Thank you! Let's leave the other repos as they are for now (and we can fix if/when there's a reason to?). And I poked at stuff but am still confused, so maybe we can look at this together some day when there's less snow.

**Comment by amandavisconti** [comment]
Need to decide whether willing to pay (new app version of Travis requires payment for over 99 builds?)

**Comment by amandavisconti** [comment]
tl;dr We either need to wait for the migration feature, or email some address below to ask them to migrate us now?
---
The following repositories cannot be migrated to travis-ci.com at this time because they are currently active on our legacy platform travis-ci.org. This feature will be available shortly. Please read our docs on open source migration to learn more.
Open source projects and their build history will continue to run on travis-ci.org for now.

However, open source repositories will be migrated to travis-ci.com gradually. The migration was planned to start at the end of Q2 2018, but has been pushed back. We will announce a new date as soon as we are able. You will receive an email when the migration for a repository is complete. This is an opt-in process: to have a repository migrated over, it must first be activated on travis-ci.com.

Currently, open source repositories that were private in any time in their history may be added to travis-ci.com. The option to migrate to travis-ci.com for active open source repositories that were always open source will come soon.

Repositories may also be migrated without their build history or build settings (including environment variables) immediately.

Please contact support support@travis-ci.com. In your email, please include:

“Repository Migration” somewhere in the subject line, and
the repository’s name in your email

**Comment by amandavisconti** [comment]
They've activated stuff for us:
"I have activated our new (beta) migration feature for your account scholarslab/scholarslab.org hope this helps!
To help you start migrating your open source repositories to travis-ci.com, we have created this guide https://docs.travis-ci.com/user/open-source-repository-migration/ that contains the instructions to transfer a project as well as some other details we thought might be useful, including what it looks like to work with a repository once it's been migrated."

**Comment by amandavisconti** [comment]
Reopening as going through the steps above did not result in Travis appearing in our PR checks again

**Comment by amandavisconti** [comment]
@walshbr @jeremyboggs Do you remember what ways Travis was useful for us? Is it still worth adding, given we've been functioning without it for a while?

**Comment by walshbr** [comment]
We mainly just used it as a deadlink checker. That's useful for link rot and also for making sure that new posts or features are functioning as intended. But that's not strictly necessary I suppose. If we take the link checker out of our rake task (travis still runs if you run the rake command locally) it will greatly speed up site builds.

**Comment by amandavisconti** [comment]
Updated issue title to reflect current needs
